doc/release/1.17.0-notes.rst

```diff
@@ -156,6 +156,11 @@ thereby saving a level of indentation
 ----------------------------------------------
 In some cases where ``np.interp`` would previously return ``np.nan``, it now
 returns an appropriate infinity.
+Specialized ``np.isnan``, ``np.isinf``, and ``np.isfinite`` ufuncs for bool and int types
```

Comment:
Nit: bad merge here, needs an extra line above

Reply:
This is fixed